contracts/DegenLockToken.sol

```diff
@@ -103,7 +110,8 @@ contract DegenLockToken is ERC20, Ownable, ReentrancyGuard {
         _mint(msg.sender, amount);
 
         depositTimestamps[msg.sender] = block.timestamp;
-        emit DepositTimestampUpdated(msg.sender, block.timestamp);
+        /* we can remove the line below to save gas */
+        emit DepositTimestampUpdated(msg.sender, block.timestamp); 
```

Comment:
We're looking to follow best practices by emitting events for variable state changes. This will be super helpful for analyzing events on platforms like Dune Analytics.